README.md

```diff
@@ -53,6 +53,37 @@ The `blessed` renderer will render a table inside a screen, that has be exited b
 
 ![](./.github/blessed.png?raw=true)
 
+### JSON
+The `JSON` renderer will print a JSON array with a match info for each package
```

Comment:
Not sure about the wording on this... does the term 'a match info' make sense here? Can we just have something like 'the match information'?